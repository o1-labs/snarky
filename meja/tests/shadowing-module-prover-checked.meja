module M = {};

module T = {
  Prover {
    module M = {};
  };
};

open T;

module M' = M;
