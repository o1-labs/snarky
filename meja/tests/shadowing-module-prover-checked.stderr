File "tests/shadowing-module-prover-checked.meja", line 11, characters 12-13:
Error: Could not find a module M:
       The current definition is only available in prover mode.
